phalcon/Storage/Adapter/Stream.zep

```diff
@@ -167,8 +167,9 @@ class Stream extends AbstractAdapter
     }
 
     /**
-     * Returns the already connected adapter or connects to the Memcached
-     * server(s)
+     * Always returns null
+     *
+     * @return null
```

Comment:
```suggestion
     * @return string
```

Reply:
@Jeckerson  Not really. It always NULL for this class